CODIGO/clsGrapchicalInventory.cls

```diff
@@ -1031,7 +1031,10 @@ Public Function GetActiveWeaponSlot() As Integer
 End Function
 
 Public Function CanUse(ByVal Slot As Integer) As Boolean
-    CanUse = UserInventory(Slot).ObjType = eObjType.otWeapon Or UserInventory(Slot).ObjType = eObjType.otPociones Or UserInventory(Slot).ObjType = eObjType.OtHerramientas Or UserInventory(Slot).ObjType = eObjType.otInstrumentos Or UserInventory(Slot).ObjType = eObjType.otNudillos Or UserInventory(Slot).ObjType = eObjType.OtCofre
+    CanUse = UserInventory(Slot).ObjType = eObjType.otWeapon Or UserInventory(Slot).ObjType = eObjType.otPociones Or _
```

Comment:
`
with UserInventory(Slot)
CanUse = .ObjType = eObjType.otWeapon Or U..ObjType = eObjType.otPociones Or _
UserInventory(Slot).ObjType = eObjType.OtHerramientas Or UserInventory(Slot).ObjType = eObjType.otInstrumentos Or _
             ObjType = eObjType.otNudillos Or ).ObjType = eObjType.OtCofre Or _
             (ObjData(UserInventory(Slot).ObjIndex).CDType > 0 And ObjData(.ObjIndex).CDType < e_CdTypes.CDCount)